commons/connector/altoida/AltoidaAcceleration.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "radarcns.connector.altoida",
+  "type": "record",
+  "name": "AltoidaAcceleration",
+  "doc": "Data from 3-axis accelerometer sensor with gravitational constant g as unit.",
+  "fields": [
+    {"name": "time", "type": "double", "doc": "Timestamp in unix time acquired locally from the device."},
+    {"name": "timeRecieved", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+    {"name": "X", "type": "float", "doc": "Acceleration in the x-axis (g)."},
```

Comment:
please follow conventions of lower camel case for naming. That means
`x`, `pitch`, `objectAction`